src/chainparams.cpp

```diff
@@ -406,7 +406,7 @@ class CRegTestParams : public CTestNetParams
 
         vFixedSeeds.clear(); //! Testnet mode doesn't have any fixed seeds.
         vSeeds.clear();      //! Testnet mode doesn't have any DNS seeds.
-
+        nExtCoinType = 1;
```

Comment:
No need to redefine it for `CRegTestParams` as it inherits from `CTestNetParams`